cypress/support/index.js

```diff
@@ -1,3 +1,11 @@
 Cypress.Screenshot.defaults({
   screenshotOnRunFailure: true,
 });
+
+// eslint-disable-next-line import/prefer-default-export
+export const el = {
+  viewport: 'viewport',
+  header: 'header',
+  section: 'section',
+  cookieBanner: 'div#root > div[class^="Wrapper"]',
```

Comment:
`'div#root > div[class^="Wrapper"]'` isn't a reliable hook as 'Wrapper' can change. Article has a better way I think https://github.com/bbc/simorgh/blob/new-frontpage-e2e/cypress/support/bodyTestHelper.js#L116

Reply:
That's good to know for the future, cheers. I've removed this from the test since the cookie stuff is covered by the cookieBannerWS test.